src/compile-cache.js

```diff
@@ -86,29 +86,30 @@ export default class CompileCache {
    * @property {string} code  The source code if the file was a text file
    * @property {Buffer} binaryData  The file if it was a binary file
    * @property {string} mimeType  The MIME type saved in the cache.
-   * @property {string[]} dependentFiles  The dependent files returned from 
+   * @property {string[]} dependentFiles  The dependent files returned from
    *                                      compiling the file, if any.
-   */   
+   */
   async get(filePath) {
-    d(`Fetching ${filePath} from cache`);
+    //d(`Fetching ${filePath} from cache`);
```

Comment:
That was not intentional, that was me debugging stuff and getting spammed, and it got lost in the whitespace changes.